api/envoy/config/metrics/v2/stats.proto

```diff
@@ -58,6 +59,30 @@ message StatsConfig {
   google.protobuf.BoolValue use_all_default_tags = 2;
 }
 
+// Inclusion/exclusion filter for stat name creation. If not provided, all stats are instantiated as
```

Comment:
Can you provide some more color here on what this actually does and why someone would want to do this?

Reply:
Done in bb48b37.